Components/LandingFooter.tsx

```diff
@@ -28,7 +28,7 @@ const LandingFooter = () => {
                 <Link href="/OurServices" className={hoverEffect}>Our Services</Link>
                 </li>
                 <li className="leading-8 text-base text-black text-opacity-80 dark:text-zinc-50">
-                <Link href="/" className={hoverEffect}>Benefits</Link>
+                <Link href="/WhyChooseUs" className={hoverEffect}>Benefits</Link>
```

Comment:
We use not use `cameCase` in the urls.
```suggestion
                <Link href="/why-choose-us" className={hoverEffect}>Benefits</Link>
```